### ISSUE TITLE
Does base plugin workflow-job need upgrading to 2.39 to support pipeline-model-extensions 1.7.1

### ISSUE THREAD

**Issue by neiljfwhite** [jenkinsci/kubernetes-operator#437]
# Expected Behavior
Jenkins Instance starts sucessfully

# Actual Behavior
Jenkins instance fails to start with error:
```
2020-07-13 11:03:59.358+0000 [id=28]	SEVERE	jenkins.InitReactorRunner$1#onTaskFailed: Failed Loading plugin Pipeline: Declarative Extension Points API v1.7.1 (pipeline-model-extensions)
java.io.IOException: Failed to load: Pipeline: Declarative Extension Points API (1.7.1)
 - Update required: Pipeline: Job (2.38) to be updated to 2.39 or higher
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:934)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:548)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:296)
	at jenkins.model.Jenkins$5.runTask(Jenkins.java:1131)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:214)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```
[pipeline-model-definition-plugin 1.7.1](https://github.com/jenkinsci/pipeline-model-definition-plugin/releases) was released on the 10th July and is being pulled in/upgraded somewhere (I have not had time to confim yet)

We haved worked around the issue by adding the `basePlugins` to our Jenkins instance manifest:

```
    basePlugins:
      - name: kubernetes
        version: 1.25.2
      - name: workflow-job
        version: "2.39"
      - name: workflow-aggregator
        version: "2.6"
      - name: git
        version: 4.2.2
      - name: job-dsl
        version: "1.77"
      - name: configuration-as-code
        version: "1.38"
      - name: kubernetes-credentials-provider
        version: "0.13"

```

# Steps to Reproduce the Problem

1. Deploy operator -> https://jenkinsci.github.io/docs/installation/
2. Create Jenkins -> https://jenkinsci.github.io/kubernetes-operator/docs/getting-started/latest/deploy-jenkins/
3. Jenkins fails to start with mentioned error.

# Additional Info
Jenkins image: `image: jenkins/jenkins:lts`

https://github.com/jenkinsci/kubernetes-operator/blob/43dffbd5d36a6f0f020ebefb0e23e8e610b4e55c/pkg/plugins/base_plugins.go#L10

- Kubernetes version:

  **Output of `kubectl version`:**

  ```
  1.16.3
  ```

- Jenkins Operator version:

```
 0.4.0
  ```

**Comment by pniederlag** [comment]
confirmed, struck by the same problem today

**Comment by neiljfwhite** [comment]
raised the simple PR above - not sure if more investigation wants to be done by the main contributors though

**Comment by smolin** [comment]
I think the workaround basePlugins block is missing a version for the final item in the list. This seemed to work for me:
`
apiVersion: jenkins.io/v1alpha2
kind: Jenkins
metadata:
  name: jenkins
  namespace: jenkins01
spec:
    master:
      basePlugins:
        - name: kubernetes
          version: 1.25.2
        - name: workflow-job
          version: "2.39"
        - name: workflow-aggregator
          version: "2.6"
        - name: git
          version: 4.2.2
        - name: job-dsl
          version: "1.77"
        - name: configuration-as-code
          version: "1.38"
        - name: kubernetes-credentials-provider
          version: "0.13"
    containers:
[...]
`

**Comment by neiljfwhite** [comment]
Good spot @smolin - cut and paste issue - comment updated

**Comment by tomaszsek** [comment]
Fixed in #442
